### ISSUE TITLE
Implement server ID argument

### ISSUE THREAD

**Issue by fl0wfr** [jeanralphaviles/prometheus_speedtest#22]
Hi,

First of all, thanks for the development of this useful module!
I noticed in my case, that the default Speedtest server used for the test is giving random results, not reflecting my broadband bandwidth reality. speedtest-cli has an option to force the test to run using a specific server:
```
  --server SERVER       Specify a server ID to test against. Can be supplied
                        multiple times
```
Could you implement this option into prometheus_speedtest?

Thanks!
Best,

**Comment by jeanralphaviles** [comment]
Just realized that this was fixed in #21, back in April. There's a `--servers` flag.

Example:

```
$ prometheus_speedtest --servers 6215,30449
```

**Comment by fl0wfr** [comment]
Thank you for this feature.
